src/hooks/useAccordionAnimation.ts

```diff
@@ -0,0 +1,21 @@
+import {useEffect} from 'react';
+import {useSharedValue} from 'react-native-reanimated';
+
+const useAccordionAnimation = (isExpanded: boolean) => {
```

Comment:
Let's use named function instead of arrow function (for consistency with other functions/hooks)